### DISCUSSION TITLE
Cannot set password for redis sentinel

### DISCUSSION THREAD

**Discussion by vipmarwah** [OT-CONTAINER-KIT/redis-operator#582] [General]
Hi,
I followed the instruction to set redis-secret but could still connect without password. 
![image](https://github.com/OT-CONTAINER-KIT/redis-operator/assets/101658736/2ebf221e-75ae-4d3e-bc86-6e3b263c89b5)
 
 wondering if there are any additional configurations required to set password for redis-sentinel. It works fine with redis-replication.  Any help will be appreciated. 
 
Thanks

**Comment by shubham-cmyk** [comment]
which redis image you are using ? 
@vipmarwah

**Reply by vipmarwah** [reply]
Thanks for the prompt reply 

quay.io/opstree/redis-sentinel:v7.0.7

**Reply by shubham-cmyk** [reply]
Can you try the latest `v7.0.12`

**Reply by vipmarwah** [reply]
sure thanks.

**Reply by shubham-cmyk** [reply]
From v7.0.12 it should support the password along with the TLS. 
https://github.com/OT-CONTAINER-KIT/redis/pull/35/files#diff-c3c0dcf3892b7d0c37d75bd5bf1865954c6839821f93b27c7083f9cd97011eb6R32

**Reply by vipmarwah** [reply]
Hi,

Just applied 
Helm Redis Operator version: 0.15.0 with Sentinel CRD  ( sentinel image : 7.0.12 ) 

Using configuration below:

apiVersion: redis.redis.opstreelabs.in/v1beta1
kind: RedisSentinel
metadata:
  name: redis-sentinel
  namespace: myredis-opr
  annotations:
    RedisReplication.opstreelabs.in/skip-reconcile: 'true'
spec:
  clusterSize: 3
  redisSentinelConfig:
    redisReplicationName: redis-sentinel
  kubernetesConfig:
    image: quay.io/opstree/redis-sentinel:v7.0.12
    imagePullPolicy: IfNotPresent
    redisSecret:
      name: sentinel-secret
      key: password

result: Pods stopped working with the error below

Warning  BackOff  92s (x222 over 46m)  kubelet  Back-off restarting failed container redis-sentinel-sentinel in pod redis-sentinel-sentinel-0_myredis-opr(abcf73fa-f4bc-4287-881c-7d8766aee1c9).

Screen Shot

![image](https://github.com/OT-CONTAINER-KIT/redis-operator/assets/101658736/e60bad13-5bef-409a-b435-8966e86ab085)

thanks in advance
